softlayer/cloud-config.yml

```diff
@@ -0,0 +1,51 @@
+azs:
+- name: z1
```

Comment:
should have 3 azs even if as a starter they are all the same. makes default example manifests work much better.